src/Components/pages/Home.jsx

```diff
@@ -1,45 +1,75 @@
 import React, {Component,Fragment} from 'react';
 import {Helmet} from 'react-helmet';
-import {Link, withRouter} from 'react-router-dom';
-
-import UserProvider from "../authenticate/providers/UserProvider";
-import { UserContext } from "../authenticate/providers/UserProvider";
-
-import { BrowserRouter as Router, Route } from 'react-router-dom';
+import {Link} from 'react-router-dom';
 
 class Home extends Component {
+    
     constructor(props) {
         super(props);
-        this.state={};
+        this.state ={user : 'N/A'};
+    }
+    
+    componentDidMount() 
+    {
+        let userAuth = null;
+        if( localStorage.getItem('User') == 'N/A')
+            userAuth = 'N/A'
+        else
+            userAuth = localStorage.getItem('User')
+        window.addEventListener("storage", e =>{
+            let change = localStorage.getItem('User')
+            this.setState({ user: change});
+        });
+        this.setState({user:userAuth})
+        const {gameId} = this.state;
+    }
+
+    getAuth = () => {
+        const user = this.state.user;
+        let auth = [];
+        if(user == 'N/A') {
+            let link =<div className="login-button-container"><p><Link to="/auth/signIn">Login</Link></p></div> ;
+            auth.push(link);
+            link = <div className="signup-button-container"><p><Link to="/auth/signUp">Sign Up</Link></p></div>;
```

Comment:
same comment as before about not needing a sign up page

Reply:
Read above.